contrib/drivers/filesharedrivers/nfs/nfs.go

```diff
@@ -86,40 +85,39 @@ func (*Driver) Unset() error { return nil }
 
 func (d *Driver) CreateFileShareAcl(opt *pb.CreateFileShareAclOpts) (fshare *model.FileShareAclSpec, err error) {
 	var access string
-	var accessTo []string
 	var accessCapability []string
 	// Get accessto list
-	accessTo = opt.GetAccessTo()
+	accessTo := opt.GetAccessTo()
 	// get accessCapability list
 	accessCapability = opt.GetAccessCapability()
 	// get fileshare name
-	fname := opt.GetMetadata()[KFileshareName]
+	fname := opt.Name
+
+	permissions := []string{"read", "write"}
 
 	for _, value := range accessCapability {
-		if value == "w" {
-			access = AccessLevelRw
-			break
-		} else {
-			access = AccessLevelRo
+		value = strings.ToLower(value)
+		if (utils.Contains(permissions, value)){
+				access += value[:1]
```

Comment:
The code style is not correct, please run `make goimports` to format it.

Reply:
Done